### DISCUSSION TITLE
Feature Request: support jump to a line according to #L line number

### DISCUSSION THREAD

**Discussion by Cluas** [community/community#2808] [Mobile]
The web supports this feature.
for example:[https://github.com/golang/go/blob/921ceadd2997f2c0267455e13f909df044234805/src/runtime/mheap.go#L358](https://github.com/golang/go/blob/921ceadd2997f2c0267455e13f909df044234805/src/runtime/mheap.go#L358)
![image](https://user-images.githubusercontent.com/10056928/100251623-edacb880-2f79-11eb-8d25-e578892b15b1.png)

But when I open this url in github app, it has no positioning behavior.
Can you add this feature？

**Comment by candyho** [comment]
Hey thanks for sharing your feedback! Cross-referencing [related post](https://github.com/github/feedback/discussions/5) here to better keep track of similar request. We will keep you posted once we land on a solution for jumping to line.

**Reply by Cluas** [reply]
> Hey thanks for sharing your feedback! Cross-referencing [related post](https://github.com/github/feedback/discussions/5) here to better keep track of similar request. We will keep you posted once we land on a solution for jumping to line.

Is this feature in plan？😳
